persistence/src/androidMain/kotlin/com/wire/kalium/persistence/db/UserDatabase.kt

```diff
@@ -35,17 +35,19 @@ fun userDatabaseBuilder(
     val dbName = FileNameUtil.userDBName(userId)
 
     val driver: AndroidSqliteDriver = if (encrypt) {
+        System.loadLibrary("sqlcipher")
         AndroidSqliteDriver(
             schema = UserDatabase.Schema,
             context = context,
             name = dbName,
-            factory = SupportFactory(passphrase.value)
+            factory = SupportOpenHelperFactory(passphrase.value, null, true)
```

Comment:
Same here, `true` here is for `enableWriteAheadLogging`. But now for the Global database.

Reply:
and here is the local database, am I right?

Reply:
What do you mean by `local` database?

Reply:
in the comment you mentioned `But now for the Global database.` I think you mean userDatabase? right?